### PR TITLE
Uavcan v1.0 bluesky

### PR DESCRIPTION
**Pull request by thirtytwobits**
Some fixes as I work on CI builds.

### CODE REVIEW

File: test/ontarget/S32K146EVB/src/LPUART.c [hunk 1]
```diff
@@ -42,10 +42,9 @@ void LPUART1_init(void)  /* Init. summary: 9600 baud, 1 stop bit, 8 bit format,
 	PCC->PCCn[PCC_LPUART1_INDEX] |= PCC_PCCn_PCS(0x02)    /* Clock Src = 2 (SIRCDIV2_CLK) */
                             	 |  PCC_PCCn_CGC_MASK;     /* Enable clock for LPUART1 regs */
 
-	// TODO: 115200
-	LPUART1->BAUD = LPUART_BAUD_SBR(0x34)  	/* Initialize for 9600 baud, 1 stop: */
-                	|LPUART_BAUD_OSR(15);  	/* SBR=52 (0x34): baud divisor = 8M/9600/16 = ~52 */
-											/* OSR=15: Over sampling ratio = 15+1=16 */
```
Comment (pavel-kirienko) [inline test/ontarget/S32K146EVB/src/LPUART.c]: image

Comment (thirtytwobits) [reply test/ontarget/S32K146EVB/src/LPUART.c]: Oh geez NXP. I'll fix this. Sorry (I didn't do it!)

File: test/ontarget/S32K146EVB/include/system_S32K146.h [hunk 2]
```diff
@@ -39,12 +38,13 @@
  */
 
 #ifndef SYSTEM_S32K146_H_
-#define SYSTEM_S32K146_H_                        /**< Symbol preventing repeated inclusion */
+#define SYSTEM_S32K146_H_ /**< Symbol preventing repeated inclusion */
```
Comment (pavel-kirienko) [inline test/ontarget/S32K146EVB/include/system_S32K146.h]: There should be two spaces before the comment. Perhaps clang-format is misconfigured?

Comment (thirtytwobits) [reply test/ontarget/S32K146EVB/include/system_S32K146.h]: Hmm. Let me see if this is something clang-format can do...

Comment (pavel-kirienko) [reply test/ontarget/S32K146EVB/include/system_S32K146.h]: It definitely can (at least v8.0 can).

Comment (thirtytwobits) [reply test/ontarget/S32K146EVB/include/system_S32K146.h]:
```
SpacesBeforeTrailingComments: 2
```
Is already in the config. I'm not sure why it didn't honor it for this line?

Comment (thirtytwobits) [reply test/ontarget/S32K146EVB/include/system_S32K146.h]: Ah, I see it now. This only works for // comments.

Comment (pavel-kirienko) [reply test/ontarget/S32K146EVB/include/system_S32K146.h]: Sad.

Comment (thirtytwobits) [reply test/ontarget/S32K146EVB/include/system_S32K146.h]: I tried to manually clean this up a bit. It is just test code. I'll be more fastidious in the libuavcan headers.

Comment (pavel-kirienko) [reply test/ontarget/S32K146EVB/include/system_S32K146.h]: Okay

**Pull request merged**